http/handler.go

```diff
@@ -217,6 +217,7 @@ func newRouter(handler *Handler) *mux.Router {
 	router.HandleFunc("/index/{index}/field/{field}", handler.handlePostField).Methods("POST")
 	router.HandleFunc("/index/{index}/field/{field}", handler.handleDeleteField).Methods("DELETE")
 	router.HandleFunc("/index/{index}/field/{field}/import", handler.handlePostImport).Methods("POST")
+	router.HandleFunc("/index/{index}/field/{field}/remote-available-shards/{shardID}", handler.handleDeleteRemoteAvailableShard).Methods("DELETE")
```

Comment:
We should move this to the `/internal` handler list. I think this would typically get called by a `deleteShard` function executed on a Pilosa node. I suppose there may be cases where we would want to make the shard unavailable without actually removing the data, but I feel like that would also be as a result of an internal function running (a resize event, for example).